addons/sale/wizard/payment_link_wizard.py

```diff
@@ -3,13 +3,29 @@
 
 from werkzeug import urls
 
-from odoo import api, models
+from odoo import api, fields, models
 
 
 class PaymentLinkWizard(models.TransientModel):
     _inherit = 'payment.link.wizard'
     _description = 'Generate Sales Payment Link'
 
+    amount_paid = fields.Monetary(
+        currency_field='currency_id',
+        string="Already Paid",
+    )
+    show_confirmation_message = fields.Boolean(compute="_compute_show_confirmation_message")
+    is_quotation = fields.Boolean()
+
+    @api.depends('amount')
+    def _compute_show_confirmation_message(self):
+        self.show_confirmation_message = (
+            self.amount_max != 0 and
+            self.amount == self.amount_max and
+            self.res_model == 'sale.order' and
+            self.is_quotation
```

Comment:
```suggestion
            self.amount_max and self.amount == self.amount_max and self.is_quotation
```
Why check the res_model here?

Reply:
To be clearer: when setting `is_quotation`, put the check there to be sure you are not on a `account.move`